raiden_contracts/constants.py

```diff
@@ -11,6 +13,57 @@
 CONTRACT_MONITORING_SERVICE = 'MonitoringService'
 CONTRACT_RAIDEN_SERVICE_BUNDLE = 'RaidenServiceBundle'
 
+# Deployed contract information
+# Deployed to Ropsten revival on 2018-09-03 from
+# raiden-contracts@fc1c79329a165c738fc55c3505cf801cc79872e4
+ROPSTEN_REGISTRY_ADDRESS = '0xf2a175A52Bd3c815eD7500c765bA19652AB89B30'
+ROPSTEN_DISCOVERY_ADDRESS = '0xEEADDC1667B6EBc7784721B123a6F669B69Eb9bD'
```

Comment:
Same as above: `ROPSTEN_ENDPOINT_REGISTRY_ADDRESS`?

Reply:
We are using `Discovery` in the client. The class is named like that too. But yes since we moved it here I will make it endpoint registry.